src/core/stdc/stdlib.d

```diff
@@ -212,11 +212,11 @@ size_t  wcstombs(char* s, in wchar_t* pwcs, size_t n);
 version( DigitalMars )
 {
     // See malloc comment about @trusted.
-    void* alloca(size_t size) pure; // non-standard
+    void* alloca(size_t size); // non-standard
```

Comment:
(apparently not: http://www.ibm.com/support/knowledgecenter/ssw_aix_72/com.ibm.aix.basetrf1/malloc.htm)